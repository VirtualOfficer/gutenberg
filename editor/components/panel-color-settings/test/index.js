/**
 * External dependencies
 */
import { shallow } from 'enzyme';
import { noop } from 'lodash';

/**
 * Internal dependencies
 */
import { PanelColorSettings } from '../';

describe( 'PanelColorSettings', () => {
	it( 'matches the snapshot', () => {
		const wrapper = shallow(
			<PanelColorSettings
				title="Test Title"
				colors={ [] }
				colorSettings={ [
					{
						value: '#000',
						onChange: noop,
						colorIndicatorAriaLabel: 'black',
						label: 'black color',
					},
					{
						value: '#111',
						onChange: noop,
						colorIndicatorAriaLabel: 'nearly black',
						label: 'nearly black color',
					},
				] }
			/>
		);

		expect( wrapper ).toMatchSnapshot();
	} );
} );
