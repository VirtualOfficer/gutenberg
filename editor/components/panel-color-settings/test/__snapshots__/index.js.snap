// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PanelColorSettings matches the snapshot 1`] = `
<PanelBody
  className="editor-panel-color-settings"
  title={
    <TextWithColorIndicators
      className="editor-panel-color-settings__panel-title"
      colorSettings={
        Array [
          Object {
            "colorIndicatorAriaLabel": "black",
            "label": "black color",
            "onChange": [Function],
            "value": "#000",
          },
          Object {
            "colorIndicatorAriaLabel": "nearly black",
            "label": "nearly black color",
            "onChange": [Function],
            "value": "#111",
          },
        ]
      }
    >
      Test Title
    </TextWithColorIndicators>
  }
>
  <ColorPaletteControl
    colorIndicatorAriaLabel="black"
    key="0"
    label="black color"
    onChange={[Function]}
    value="#000"
  />
  <ColorPaletteControl
    colorIndicatorAriaLabel="nearly black"
    key="1"
    label="nearly black color"
    onChange={[Function]}
    value="#111"
  />
</PanelBody>
`;
